src/vkscript/command_parser.cc

```diff
@@ -647,7 +647,13 @@ Result CommandParser::ProcessSSBO() {
       return Result("Invalid size value for ssbo command: " +
                     token->ToOriginalString());
 
-    cmd->SetSize(token->AsUint32());
+    uint32_t size = token->AsUint32();
+    cmd->SetSize(size);
+
+    // Resize the buffer so we'll correctly create the descriptor sets.
+    auto* buf = cmd->GetBuffer();
```

Comment:
Is this actually related?

Reply:
I order to make the repeat.amber script work but buffers needed to work. To make buffers work, requires the ssbo's to be sized correctly. So ... kinda?